build/android/patches/irrlicht-back_button.patch

```diff
@@ -1,14 +1,6 @@
---- irrlicht/source/Irrlicht/Android/CIrrDeviceAndroid.cpp	2014-06-03 20:56:21.289559503 +0200
-+++ irrlicht/source/Irrlicht/Android/CIrrDeviceAndroid.cpp.orig	2014-06-03 20:57:39.281556749 +0200
-@@ -423,6 +423,7 @@
- 			}
- 			
- 			device->postEventFromUser(event);
-+			status = 1;

```

Comment:
@est31, https://github.com/zaki/irrlicht/blob/96cf9444667431279e25d7a1c8c0812748e1fbb3/source/Irrlicht/Android/CIrrDeviceAndroid.cpp#L489
Just look at this. Also, Minetest use ONLY ONE hardware button - BACK, instead of Esc. Should we worry about this issue?